wiki/Beatmap_search/ru.md

```diff
@@ -16,25 +15,27 @@ tags:
 
 *См. также: [Скачивание карт](/wiki/Beatmap#скачивание)*
 
-По умолчанию всё, что вы вводите на [экране выбора песни](/wiki/Client/Interface#экран-выбора-песни) или в поисковой строке веб-сайта, будет использовано для поиска по текстовым полям карт, куда входят название исполнителя, песни, сложности, ник маппера, источник песни, а также теги карты. Если вместо текста ввести число, то поиск будет идти среди карт и сложностей с этим [идентификатором](/wiki/Beatmap#идентификация). Все запросы нечувствительны к регистру.
+По умолчанию всё, что вы вводите на [экране выбора песни](/wiki/Client/Interface#экран-выбора-песни) или в поисковой строке веб-сайта, будет использовано для поиска по текстовым полям карт, куда входят название исполнителя, песни, сложности, ник маппера, источник песни, а также [теги карты](/wiki/Beatmap/Beatmap_tags#теги,-выставляемые-маппером) и [пользовательские-теги](/wiki/Beatmap/Beatmap_tags#пользовательские-теги). Если вместо текста ввести число, то поиск будет идти среди карт и сложностей с этим [идентификатором](/wiki/Beatmap#идентификация). Все запросы нечувствительны к регистру.
```

Comment:
If we're going to link to User tags in the translation, we should also add it to the English article. As per the [osu!wiki content parity policies](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#content-parity), translations should strictly be based on the original articles!